pkg/services/sqlstore/sqlstore.go

```diff
@@ -185,6 +187,60 @@ func (ss *SQLStore) ensureMainOrgAndAdminUser() error {
 	return err
 }
 
+func (ss *SQLStore) ensureDashboardACLDefaults() error {
```

Comment:
Can we move this to dashboard_acl.go to keep dashboard acl database logic in the same file?